bundles/ranvier-commands/commands/recall.js

```diff
@@ -11,13 +11,6 @@ module.exports = srcPath => {
         return B.sayAt(player, 'You do not have a home waypoint set.');
       }
 
-      if (player.getAttribute('energy') < (player.getMaxAttribute('energy') / 3)) {
-        return B.sayAt(player, 'You do not have enough energy to recall.');
-      }
-
-      const cost = Math.round(player.getMaxAttribute('energy') / 3);
-      player.lowerAttribute('energy', cost);
```

Comment:
ha, good thing you're removing this. This was _completely_ wrong in the first place. `lowerAttribute` is a crazy dangerous method to use.